docs/content/latest/explore/ysql-language-features/going-beyond-sql/tablespaces.md

```diff
@@ -219,6 +219,61 @@ is not supported via tablespaces yet. This feature is tracked [here](https://git
 
 {{< /tip >}}
 
+## Indexes
+
+Just like tables, indexes can also be associated with a tablespace. If a table has multiple indexes, the closest index will be chosen to serve the query. For instance, lets create 3 indexes for each region occupied by the `multi_region_table` above:
```

Comment:
Suggestion:

Similarly to tables, indexes can be associated with a tablespace. If a table has more than one index, the closest index is chosen to serve the query. The following example creates three indexes for each region occupied by the  'multi_region_table' from <link to Create a multi-region table>: